### PR TITLE
Added shopping

### PR DESCRIPTION
**Pull request by lilliankravitz**
Removed shopping from collections to its own section. Updated sync and autofill

### CODE REVIEW

File: microsoft-edge/privacy-whitepaper.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Data collected by other search providers follow the privacy policy of the compan
 
 ## Autofill  
 
-Autofill in Microsoft Edge helps you be more productive by letting you save passwords, payment info, addresses, and other form entry data.  When you visit a site and start to fill out a form, Microsoft Edge uses form fill info to match your saved autofill data to the form.  Microsoft Edge offers form entry data you previously saved when you encounter similar forms.  Passwords and credit card info are only saved with your explicit permission for each password and card.  
+Autofill in Microsoft Edge helps you be more productive by letting you save passwords, payment info, addresses, and other form entry data like birthday.  When you visit a site and start to fill out a form, Microsoft Edge uses form fill info to match your saved autofill data to the form.  Microsoft Edge offers form entry data you previously saved when you encounter similar forms.  Passwords and credit card info are only saved with your explicit permission for each password and card.
```
Comment (Reezaali) [inline microsoft-edge/privacy-whitepaper.md:46]:
```suggestion
Autofill in Microsoft Edge helps you be more productive by letting you save passwords, payment information, addresses, and other form entry data, for example birthdays.  When you visit a site and start to fill out a form, Microsoft Edge uses form fill info to match your saved autofill data to the form.  Microsoft Edge offers form entry data you previously saved when you encounter similar forms.  Passwords and credit card info are only saved with your explicit permission for each password and card.  
```

File: microsoft-edge/privacy-whitepaper.md [hunk 2]
```diff
@@ -373,6 +371,20 @@ By default, your current DNS service provider is used to avoid disruptions to yo
 
 Secure DNS is off by default for managed machines that are part of an organization but it can be configured using administrative policies. InPrivate browsing uses the secure DNS setting of the profile from which the InPrivate session was launched. Guest mode will always use your current service provider.
 
+## Shopping
+
+Microsoft Edge helps you find coupons and better prices while shopping online.  To help you find coupons while shopping online, Microsoft Edge downloads a list of shopping domains locally to the client from the Microsoft shopping service.  When you go to a website, Microsoft Edge locally determines if the site you're on is a shopping domain.  If the site is determined to be a shopping domain, Microsoft Edge will send the URL scrubbed of personally identifiable information to the Microsoft shopping service.  The shopping service will return any available coupons for that site.
```
Comment (Reezaali) [inline microsoft-edge/privacy-whitepaper.md:376]:
```suggestion
Microsoft Edge helps you find coupons and better prices while shopping online.  To help you find coupons while shopping online, Microsoft Edge downloads a list of shopping domains locally to the client from the Microsoft shopping service.  When you go to a website, Microsoft Edge determines if the site you're on is a shopping domain.  If the site is determined to be a shopping domain, Microsoft Edge will send the URL with no personally identifiable information to the Microsoft shopping service.  The shopping service will return any available coupons for that site.  
```

File: microsoft-edge/privacy-whitepaper.md [hunk 3]
```diff
@@ -373,6 +371,20 @@ By default, your current DNS service provider is used to avoid disruptions to yo
...
 
+## Shopping
+
+Microsoft Edge helps you find coupons and better prices while shopping online.  To help you find coupons while shopping online, Microsoft Edge downloads a list of shopping domains locally to the client from the Microsoft shopping service.  When you go to a website, Microsoft Edge locally determines if the site you're on is a shopping domain.  If the site is determined to be a shopping domain, Microsoft Edge will send the URL scrubbed of personally identifiable information to the Microsoft shopping service.  The shopping service will return any available coupons for that site.  
+
+When you're applying coupons, cookies will be stored on your device to correctly attribute the coupon provider.  Cookies will be saved by our trusted coupon providers only after a coupon is successfully applied on the cart.  After the coupons are applied, info about the success of the coupons will be sent back to the Microsoft shopping service to help us understand which coupons are successful and which failed.  Data sent to the Microsoft shopping service is sent over HTTPS with a randomly generated identifier which changes per coupon lookup.
```
Comment (Reezaali) [inline microsoft-edge/privacy-whitepaper.md:378]:
```suggestion
When you're applying coupons, cookies will be stored on your device to correctly attribute the coupon provider.  Cookies will be saved by our trusted coupon providers only after a coupon is successfully applied on the cart.  After the coupons are applied, information about the success of the coupons will be sent back to the Microsoft shopping service to help us understand which coupons succeeded or failed.  Data sent to the Microsoft shopping service is sent over HTTPS with a randomly generated identifier which changes per coupon lookup.  
```

File: microsoft-edge/privacy-whitepaper.md [hunk 4]
```diff
@@ -373,6 +371,20 @@ By default, your current DNS service provider is used to avoid disruptions to yo
...
+
+:::image type="complex" source="./media/whitepaper-media/shopping.png" alt-text="Shopping and coupons" lightbox="./media/whitepaper-media/shopping.png":::
+   Shopping and coupons  
+:::image-end:::  
+
+To help you find the best price while shopping online and using Collections, Microsoft Edge determines locally if the page or collection item you're viewing is a product detail page.  If you're looking at a product detail page, Microsoft Edge sends the details of the product to the shopping service, including the URL scrubbed of personally identifiable information, price, product image, product name, ratings, and reviews, along with info about Microsoft Edge and your operating system version.  This data is sent over HTTPS with a randomly generated identifier.  The Microsoft shopping service will return prices from other retailers for the product you're viewing.
```
Comment (Reezaali) [inline microsoft-edge/privacy-whitepaper.md:384]:
```suggestion
To help you find the best price while shopping online and using Collections, Microsoft Edge determines if the page or collection item you're viewing is a product detail page.  If you're looking at a product detail page, Microsoft Edge sends the details of the product to the shopping service, including the URL with no personally identifiable information, price, product image, product name, ratings, and reviews. Also, information about Microsoft Edge and your operating system version is sent to the service.  This data is sent over HTTPS with a randomly generated identifier.  The Microsoft shopping service will return prices from other retailers for the same product.  
```

File: microsoft-edge/privacy-whitepaper.md [hunk 5]
```diff
@@ -373,6 +371,20 @@ By default, your current DNS service provider is used to avoid disruptions to yo
...
+   Shopping and coupons  
+:::image-end:::  
+
+To help you find the best price while shopping online and using Collections, Microsoft Edge determines locally if the page or collection item you're viewing is a product detail page.  If you're looking at a product detail page, Microsoft Edge sends the details of the product to the shopping service, including the URL scrubbed of personally identifiable information, price, product image, product name, ratings, and reviews, along with info about Microsoft Edge and your operating system version.  This data is sent over HTTPS with a randomly generated identifier.  The Microsoft shopping service will return prices from other retailers for the product you're viewing.  
+
+The shopping service is turned on by default for all users.  You can turn off shopping in Microsoft Edge by going to `edge://settings/privacy` and turning off **Save time and money with Shopping in Microsoft Edge**.  InPrivate browsing uses the shopping setting of the profile from which the InPrivate session was launched.
```
Comment (Reezaali) [inline microsoft-edge/privacy-whitepaper.md:386]:
```suggestion
The shopping service is turned on by default for all users.  To change the shopping setting in Microsoft Edge, navigate to `edge://settings/privacy`, and then turn off **Save time and money with Shopping in Microsoft Edge**.  InPrivate browsing uses the shopping setting of the profile that launched the InPrivate session.  
```

**Review by Reezaali — CHANGES_REQUESTED** [review]

File: microsoft-edge/privacy-whitepaper.md (hunk 2 above)
Comment (lilliankravitz) [reply microsoft-edge/privacy-whitepaper.md:376]: too definitive.

File: microsoft-edge/privacy-whitepaper.md (hunk 4 above)
Comment (lilliankravitz) [reply microsoft-edge/privacy-whitepaper.md:384]: same. too definitive.

**Review by Reezaali — APPROVED** [review]

**Comment by jm-trd-ms** [comment]
#sign-off offline and merged.

**Pull request merged**